dev/com.ibm.ws.logging/src/com/ibm/ws/logging/collector/CollectorJsonUtils.java

```diff
@@ -192,8 +192,11 @@ private static String jsonifyTraceAndMessage(int maxFieldLength, String wlpUserD
                 }
 
                 else if (key.equals(LogFieldConstants.MESSAGE)) {
-
-                    String formattedValue = CollectorJsonHelpers.formatMessage(kvp.getStringValue(), maxFieldLength);
+                    String message = kvp.getStringValue();
+                    if (genData.getLogLevel().equals("ENTRY") || genData.getLogLevel().equals("EXIT")) {
```

Comment:
for getting LogLevel, for now create a new method that just iterates through all of the keys in GenericData to find the LogLevel KVP with key.equals(LogFieldConstants.LOGLEVEL) and return the corresponding value.  Fix to use LogTraceData object will eliminate that extra lookup.